extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -326,13 +309,44 @@ const deploymentTitle = computed(() => {
   return result.title;
 });
 
-const deploymentSubTitle = computed(() => {
+const deploymentSubTitles = computed(() => {
+  const subTitles: string[] = [];
+  subTitles.push(credentialSubTitle.value);
+  if (entrypointSubTitle.value) {
+    subTitles.push(entrypointSubTitle.value);
+  }
+  return subTitles;
+});
+
+const credentialSubTitle = computed(() => {
   if (home.serverCredential?.name) {
     return `${home.serverCredential.name}`;
   }
   return `Missing Credential for ${home.selectedContentRecord?.serverUrl}`;
 });
 
+const entrypointSubTitle = computed(() => {
+  if (
+    home.selectedConfiguration &&
+    !isConfigurationError(home.selectedConfiguration)
+  ) {
+    const contentRecord = home.selectedContentRecord;
+    const config = home.selectedConfiguration;
+    if (
+      contentRecord &&
+      config.configuration.entrypoint &&
+      !isConfigurationError(config)
```

Comment:
Looks like a duplicate check here. We check if the config is in error above. We could simplify by setting `const config` earlier in the function

Reply:
I refactored this to match the earlier input regarding just showing the project dir when we can't determine the entrypoint, but I believe we still need a check here (which I've changed to use the computed function that validates the config). When a configuration is in error, that is shown below the button, but we still need to show some values within the button right? Let me know if I'm not following the same path as you are, once you've looked at my changes. Thanks!